groups/bsl/bslstl/bslstl_optional.t.cpp

```diff
@@ -70,15 +69,14 @@ void aSsErT(bool condition, const char *message, int line)
 
 }  // close unnamed namespace
 
-
 // ============================================================================
 //               STANDARD BSL TEST DRIVER MACRO ABBREVIATIONS
 // ----------------------------------------------------------------------------
 
-#define ASSERT       BSLS_BSLTESTUTIL_ASSERT
-#define ASSERTV      BSLS_BSLTESTUTIL_ASSERTV
+#define ASSERT BSLS_BSLTESTUTIL_ASSERT
```

Comment:
bde-format undone.

Reply:
replying to "Did you have any success integrating bde-format with the editor?" since git doesn't let me add a comment there :/
I found that running the bde-format tool in a terminal and looking at the differences in the eclipse compare tool is a very simple way to pick and choose what gets formatted (provided I know which bits to skip, of course). Consequently, I haven't bothered with the integration.